package-res/resources/web/pentaho/visual/ccc/abstract/View.js

```diff
@@ -167,6 +169,50 @@ define([
     //endregion
 
     //region VizAPI implementation
+
+
+    _init: function() {
+      this.base();
```

Comment:
I am sure this kind of ad hoc code requires a notice that it is temporary so that you and others quickly understand what it is doing here.